internal/workload/mapping/mapping.go

```diff
@@ -67,6 +69,20 @@ func (m *MappingRepository) Remove(name string) error {
 	return m.persist()
 }
 
+func (m *MappingRepository) RemoveMappingFile() error {
+	m.lock.Lock()
+	defer m.lock.Unlock()
+
+	log.Info("Deleting workload-mapping.json file")
```

Comment:
could you replace hardcoded file name with the value of `m.mappingFilePath` so any future changes to that value will be reflected in an updated log message ?